lib/mayaUsdAPI/proxyStage.cpp

```diff
@@ -45,36 +53,40 @@ struct ProxyStageImp
 };
 
 // Use unique_ptr with custom deleter
-using UniqueProxyStageImp = std::unique_ptr<MayaUsdAPI_v0::ProxyStageImp, ProxyStageImpDeleter>;
+using UniqueProxyStageImp = std::unique_ptr<MayaUsdAPI::ProxyStageImp>;
 
 ProxyStage::ProxyStage(const MObject& obj)
 {
-    PXR_NAMESPACE_USING_DIRECTIVE
-    TF_AXIOM(!obj.isNull());
+    if (obj.isNull()) {
+        const std::string errMsg
+            = PXR_NS::TfStringPrintf(
+                  "The MObject passed to the constructor of ProxyStage is null !")
+                  .c_str();
+        throw std::runtime_error(errMsg);
```

Comment:
Why do we need to use TfStringPrintf (there are no args in this string).
```suggestion
        throw std::runtime_error("The MObject passed to the constructor of ProxyStage is null !");
```